components/datatypes/marketing/commerce.schema.json

```diff
@@ -144,6 +144,11 @@
           "$ref": "https://ns.adobe.com/xdm/datatypes/requisitionlist",
           "description": "The properties of requisition list created by customer."
         },
+        "xdm:commerce.requisitionListDeletes": {
+          "title": "Delete Requisition List",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/requisitionlist",
```

Comment:
Isn't this the field that should be a 1 when the event sent is deleteRequisitionList? So it should ref "https://ns.adobe.com/xdm/data/measure" like the "xdm:requisitionListOpens" field does (line 127)?

Reply:
If I'm understanding that correctly, I'd also personally prefer to have it located right under "requisitionListOpens" or at least under "requisitionListRemovals", rather than in the middle of the fields that don't correspond to event types